src/mutation/__tests__/RelayMutationQuery-test.js

```diff
@@ -458,6 +460,38 @@ describe('RelayMutationQuery', () => {
         .toEqualQueryNode(expected);
     });
 
+    it('warns when rangeBehaviors dont match with tracked connections', () => {
+      tracker.getTrackedChildrenForID.mockReturnValue(getNodeChildren(Relay.QL`
+        fragment on Feedback {
+          comments(orderby:"ranked_threaded",first:"10") {
+            edges {
+              node {
+                body {
+                  text
+                }
+              }
+            }
+          }
+        }
+      `));
+      var node = RelayMutationQuery.buildFragmentForEdgeInsertion({
+        fatQuery,
+        tracker,
+        connectionName: 'comments',
+        parentID: '123',
+        edgeName: 'feedbackCommentEdge',
+        parentName: 'feedback',
+        rangeBehaviors,
```

Comment:
nit: i suspect you borrowed this from other test cases in this file, but it would make the test more self-contained and readable if we were explicit about the range behaviors, e.g `rangeBehaviors: {}`.